crawler/src/main/java/com/brandwatch/ahmedanalytics/crawler/Config.java

```diff
@@ -18,4 +23,14 @@ public Twitter twitter(final @Value("${spring.social.twitter.appId}") String app
         return new TwitterTemplate(appId, appSecret);
     }
 
+    @Bean
+    public KafkaProducer<String, String> kafkaProducer() {
+        Properties props = new Properties();
+        props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "kafka:9092");
```

Comment:
Can you make this configuration `"kafka:9092"` configurable in a same way uri of mongodb is configurable?

Reply:
will give it a try

Reply:
Done